librz/bin/p/bin_bf.c

```diff
@@ -93,6 +93,10 @@ static bool check_buffer(RzBuffer *buf) {
 	return true;
 }
 
+static bool check_filename(const char *filename) {
+	return rz_str_endswith(filename, ".bf");
```

Comment:
What about making it case-insensitive?

Reply:
okay d2313fd66d69dc3f6b2b5cd5123f41adb767ca4e